src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/OnboardingSupervisor.kt

```diff
@@ -1158,6 +1163,48 @@ internal class OnboardingSupervisor(
         }
     }
 
+    private fun fetchTransferStatusSkip(
+        hash: String,
+        fromChainId: String?,
+    ) {
+        val oldState = stateMachine.state
+//        If transfer is not yet tracked, must track first before querying status
+        val isTracked = oldState?.trackStatuses?.get(hash) == true
+        if (!isTracked) {
+            trackTransferSkip(hash = hash, fromChainId = fromChainId)
+            return
+        }
+        val params: IMap<String, String> = iMapOf(
+            "tx_hash" to hash,
+            "chain_id" to fromChainId,
+        ).filterNotNull()
+        val url = helper.configs.skipV2Status()
+        helper.get(url, params) { _, response, httpCode, _ ->
+            if (response != null) {
+                update(stateMachine.squidStatus(response, hash), oldState)
```

Comment:
squidstatus?

Reply:
man, i think this is a rebase issue. this is an old PR. thanks